### PR TITLE
Minespace - variance/overview bug, nav links styling

### PR DESCRIPTION
**Pull request by mmumby**
- PR addressed the crashing bug on MineSpace, added additional check for 'transformedMineTypes'
- Added active styling for navlinks

- branch name was incorrectly named

### CODE REVIEW

File: services/minespace-web/src/components/layout/HeaderDropdown.js [hunk 1]
```diff
@@ -30,6 +31,11 @@ export class HeaderDropdown extends Component {
     signOutFromSiteMinder();
   };
 
+  setActiveLink = (pathname) => {
+    console.log(this.props.location);
```
Comment (exzizt) [inline services/minespace-web/src/components/layout/HeaderDropdown.js]: remove debug

File: services/minespace-web/src/tests/components/layout/HeaderDropdown.js [hunk 2]
```diff
@@ -12,14 +12,15 @@ const setupDispatchProps = () => {
 const setupProps = () => {
   props.keycloak = { logout: jest.fn() };
   props.isAuthenticated = true;
+  props.location = { pathname: "/mines" };
```
Comment (exzizt) [inline services/minespace-web/src/tests/components/layout/HeaderDropdown.js]: Should import routes.js and use MINES.route

File: services/minespace-web/src/components/layout/HeaderDropdown.js [hunk 3]
```diff
@@ -74,10 +80,10 @@ export class HeaderDropdown extends Component {
     return (
       <span>
         <MediaQuery minWidth={smallestDesktopWidth}>
-          <Link to={route.MINES.route} className="header-link">
+          <Link to={route.MINES.route} className={this.setActiveLink("/mines")}>
             My Mines
           </Link>
-          <Link to={route.USERS.route} className="header-link">
+          <Link to={route.USERS.route} className={this.setActiveLink("/users")}>
```
Comment (exzizt) [inline services/minespace-web/src/components/layout/HeaderDropdown.js]: Should import routes and use MINES.route and USERS.route respectively

Comment (mmumby) [reply services/minespace-web/src/components/layout/HeaderDropdown.js]: yes good catch

File: services/minespace-web/src/components/layout/HeaderDropdown.js (hunk 1 above)
Comment (mmumby) [reply services/minespace-web/src/components/layout/HeaderDropdown.js]: gone!

**Pull request merged**